llvm/lib/Target/RISCV/RISCVInstrInfoXCheri.td

```diff
@@ -381,25 +388,44 @@ let defsCanBeSealed = 1 in
 def CSeal           : Cheri_rr<0xb, "cseal", GPCR, GPCR>;
 def CUnseal         : Cheri_rr<0xc, "cunseal", GPCR, GPCR>;
 def CAndPerm        : Cheri_rr<0xd, "candperm">;
+def : MnemonicAlias<"acperm", "candperm">;
 def CSetFlags       : Cheri_rr<0xe, "csetflags">;
+// NB: No alias for scmode->csetflags since the argument is inverted.
 def CSetOffset      : Cheri_rr<0xf, "csetoffset">;
 def CSetAddr        : Cheri_rr<0x10, "csetaddr">;
+def : MnemonicAlias<"scaddr", "csetaddr">;
 def CSetHigh        : Cheri_rr<0x16, "csethigh">;
+def : MnemonicAlias<"schi", "csethigh">;
 let isReMaterializable = 1, isAsCheapAsAMove = 1 in
 def CIncOffset      : Cheri_rr<0x11, "cincoffset">;
+def : MnemonicAlias<"cadd", "cincoffset">;
```

Comment:
This is meant to be just add though right? (Possibly with cadd as an alternative mnemonic?)

Reply:
Oh yeah good catch! Thinking about it a bit more I'm not sure we need the "c"-prefixed alternatives since the operand uniquely identifies the operation.